stock_picking_line_sequence/models/stock.py

```diff
@@ -84,8 +84,10 @@ def _reset_sequence(self):
         for rec in self:
             current_sequence = 1
             for line in rec.move_ids_without_package:
-                line.sequence = current_sequence
-                current_sequence += 1
+                # Check if the record ID is an integer (real ID) or a string (virtual ID)
+                if isinstance(line.id, int):
```

Comment:
@bosd FYI, usually, we use NewId class instead.

So:

`if not isinstance(line.id, NewId):`